runtime/executor/method.cpp

```diff
@@ -805,6 +805,21 @@ Error Method::init(
       pte_data_map = pte_data_map_res.get();
     }
 
+    int named_data_map_items =
+        (named_data_map != nullptr) ? named_data_map->get_num_keys().get() : 0;
+    int pte_map_items =
+        (pte_data_map != nullptr) ? pte_data_map->get_num_keys().get() : 0;
+
+    // TODO: implement ndm merge.
+    ET_CHECK_OR_RETURN_ERROR(
+        (pte_map_items == 0 || named_data_map_items == 0),
+        NotImplemented,
+        "NamedDataMap merge not supported. Both pte_data_map and named_data_map are non-empty, don't know what to do.");
```

Comment:
nit: remove the dont know what to do part haha. Direct people seeing this crash to open an issue